### ISSUE TITLE
Tabs plugin not functioning at all

### ISSUE THREAD

**Issue by bobmhughes** [pengx17/logseq-plugin-tabs#61]
This is a great plug-in, in fact I can't imaging using Logseq without it, however for some unknown reason, it has stopped functioning on both my Windows & Mac computers. I have the latest version of logseq and the Tabs plug in installed. The first file I click on in recent or favorites opens in a tab, and subsequent files that I click in are not added too open files, but replace the file that was last opened. Any help would be appreciated. Thanks

**Comment by pengx17** [comment]
> The first file I click on in recent or favorites opens in a tab, and subsequent files that I click in are not added too open files, but replace the file that was last opened. 

@bobmhughes I think this is the default behavior. Opening new page action will not add new tabs, but replace the current tab. You will need to hold ctrl (or CMD) key whiling clicking the page link ref.

**Comment by bobmhughes** [comment]
Thank you for your response that was most helpful.

When I hold down the ctrl key while clicking the page ref link, the corresponding page will indeed open. However, unless I return to the Daily Journal first, subsequent attempts to open additional pages by holding down the ctrl key while clicking the page ref link,  the new page opened will replace the last tab/page opened. I don’t know if this is intended behavior of the plug-in or not, but if it is required to return to the daily journal to open additional pages, then I must conclude that the last additional page opened has the cursor focused on it, and therefore is replaced when opening yet an another page.

Thanks again,

Bob


From: Peng Xiao ***@***.***>
Date: Friday, October 7, 2022 at 1:25 AM
To: pengx17/logseq-plugin-tabs ***@***.***>
Cc: bobmhughes ***@***.***>, Mention ***@***.***>
Subject: Re: [pengx17/logseq-plugin-tabs] Tabs plugin not functioning at all (Issue #61)

The first file I click on in recent or favorites opens in a tab, and subsequent files that I click in are not added too open files, but replace the file that was last opened.

@bobmhughes<https://github.com/bobmhughes> I think this is the default behavior. Opening new page action will not add new tabs, but replace the current tab. You will need to hold ctrl (or CMD) key whiling clicking the page link ref.

**Comment by pengx17** [comment]
> if it is required to return to the daily journal to open additional pages
This sounds like a bug indeed. The plugin should work the same no matter if it is in the journal pages or in a page. Do you have any other plugins that are enabled? If yes, please try disable others, reload the app and try if the plugin still not function well?

**Comment by bobmhughes** [comment]
I do have other plugins installed, so I did as you suggested and disabled all of them. However, at least on my MacBook Pro M1 running MacOS 12.6 and Logseq 0.8.8 and Tabs 1.17.2, even with all other plugins disabled, I still must return to the Daily Journal between opening more pages on new tabs.

Thanks for your help.

Bob



From: Peng Xiao ***@***.***>
Date: Friday, October 7, 2022 at 8:13 PM
To: pengx17/logseq-plugin-tabs ***@***.***>
Cc: bobmhughes ***@***.***>, Mention ***@***.***>
Subject: Re: [pengx17/logseq-plugin-tabs] Tabs plugin not functioning at all (Issue #61)

if it is required to return to the daily journal to open additional pages
This sounds like a bug indeed. The plugin should work the same no matter if it is in the journal pages or in a page. Do you have any other plugins that are enabled? If yes, please try disable others, reload the app and try if the plugin still not function well?

**Comment by shkarbon** [comment]
I can confirm this is happening as described for me as well. I also tried to disable plugins but the issue was still there. 

Other plugins i have are

Awesome UIv1.3.8
Automatic format url titlev1.1.0
Journals calendar0.10.5
Sticky Headersv1.4.2
Awesome Linksv1.10.5
GPT-3 OpenAIv1.0.1
Tabsv1.17.2
Block to pagev1.3.0
Link Previewv2.2.5
logseq-wordcount-pluginv1.8

**Comment by hiltronrex** [comment]
I can confirm this is happening as described for me as well. I also tried to disable plugins but the issue was still there.

Other plugins i have are

Awesome UIv1.3.8
Automatic format url titlev1.1.0
Journals calendar0.10.5
Sticky Headersv1.4.2
Awesome Linksv1.10.5
GPT-3 OpenAIv1.0.1
Tabsv1.17.2
Block to pagev1.3.0
Link Previewv2.2.5
logseq-wordcount-pluginv1.8

**Comment by pengx17** [comment]
🤔 I guess it is time to set up an issue reporter in the plugin ... Did you relaunch the app after disabling all other plugins?

**Comment by hiltronrex** [comment]
Yes. Tried that and it didn't work. Here is a gif of me being able to open pages from journal but not from a page.

![Kapture 2022-10-24 at 16 39 45](https://user-images.githubusercontent.com/16205620/197455598-ff4de778-99ce-41bb-9699-254895ca96de.gif)

**Comment by richardchui** [comment]
This is happening to me as well.  Tabs has been working great until a few days ago.  Pressing Cmd-Click will replace the content of the current tab.  It is only when the Daily Journal is displayed then a new tab will be opened.

**Comment by pengx17** [comment]
@richardchui can you please share the list of installed plugins/themes, and custom.css/custom.js you are using if there is any?

**Comment by richardchui** [comment]
Plugins - Logseq Find and Replace, Journal Calendar, Logseq Markmap, Tags, Tabs
Themes - Dev theme
No custom css/js

iMac running Ventura

**Comment by richardchui** [comment]
1.18.1 seems to have fixed the issue.  Thanks!!

**Comment by pengx17** [comment]
@richardchui great! Actually I am not sure why it fixed your issue ... 😅

**Comment by richardchui** [comment]
I was wrong.  It works only right after I restarted Logseq.  Issue has not been solved...

**Comment by jwwang0-0** [comment]
I had the same issue before. But after updating all my plugins to the newest version, it is working now. Maybe that helps?

**Comment by feng1o** [comment]
this is happening to me as well. version :  v1.19.1
 installed plugins: 

> logseq-agenda/
logseq-awesome-links/
logseq-awesome-styler/
logseq-banners-plugin/
logseq-block-calendar/
logseq-bonofix-theme/
logseq-bullet-threading/
logseq-calendars-plugin/
logseq-dev-theme/
logseq-doc/
logseq-heatmap/
logseq-journals-calendar/
logseq-kanban-plugin/
logseq-link-preview/
logseq-mark-map/
logseq-markdown-table/
logseq-pdf-export-plugin/
logseq-plugin-gpt3-openai/
logseq-random-note/
logseq-tabs/
logseq-tags/
logseq-tocgen/
logseq-todo-master/
logseq-todo-plugin/
logseq-vim-shortcuts/
logseq-wrap/


https://github.com/pengx17/logseq-plugin-tabs/assets/10048132/c5a7d246-be6c-4ee7-b1e9-0686ccffab8f
